### PR TITLE
Update traditional to salt migration documentation

### PR DESCRIPTION
**Pull request by vandabarata**
# Some hints

Consider adding an entry to the `CHANGELOG.md` file in the toplevel directory.
Cosmetic changes such as fixing typos do not need log entries (nevertheless it is important to fix typos, etc.)!
In the `manager-4.3`, the hidden `.changelog` file with a leading dot is still in use.

Add Description, Target branches, and related Links below the following section titles.

In the Description, just enter a summary of why you created this PR (and, if available, add any relevant diagram).

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 4.3, SUMA MU X.Y.Z, or SUMA development version).  This information can be helpful if `ifeval` statements are needed to publish it for certain products only.
4.3

Backport targets:
- No backports should be needed, only applicable to 4.3

# Links
- This PR tracks issue https://github.com/SUSE/spacewalk/issues/22391
- Related development PR #<insert PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
Thanks, Vanda, looks great!  With minimal cleanup, we can merge it.

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 1]
```diff
@@ -3,87 +3,37 @@
...
-* create a bootstrap script,
-* modify the bootstrap script, and
-* run the bootstrap script to register the client.
+This could be done by:
 
+* generating a reactivation key for the client,
+* creating a bootstrap script with a specific activation key, and
+* running the bootstrap script to register the client, with the aforementioned reactivation key.
 
+== Generate reactivation key
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:12]:
```suggestion


== Generate Reactivation Key
```

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 2]
```diff
@@ -3,87 +3,37 @@
...
+A reactivation key can be used to re-register the client and regain all {productname} settings. For more information on how to create one, refer to xref:client-configuration/activation-keys.adoc#_reactivation_keys[]
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:13]:
```suggestion
A reactivation key can be used to re-register the client and regain all {productname} settings. 
For more information on how to create reactivation key, see xref:client-configuration/activation-keys.adoc#_reactivation_keys[]
```

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 3]
```diff
@@ -3,87 +3,37 @@
...
 
 == Create bootstrap script
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:15]:
```suggestion


== Create Bootstrap Script
```

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 4]
```diff
@@ -3,87 +3,37 @@
...
-
-First you create a bootstrap script with the {webui} as a template.
-
 .Procedure: Creating a bootstrap script
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:16]:
```suggestion
.Procedure: Creating Bootstrap Script
```

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 5]
```diff
@@ -3,87 +3,37 @@
...
-. In the {productname} {webui}, navigate to menu:Admin[Manager Configuration > Bootstrap Script].
-. In the [guimenu]``SUSE Manager Configuration - Bootstrap`` dialog, verify that the [guimenu]``Bootstrap using Salt`` checkbox is checked.
-. The required fields are pre-populated with values derived from previous installation steps.
-  For details on each setting, see xref:reference:admin/bootstrap-script.adoc[].
-. Click btn:[Update] to create the script.
-. The bootstrap script is generated and stored on the server in the [path]``/srv/www/htdocs/pub/bootstrap`` directory.
+A bootstrap script can be created through the WebUI or from the command line. Both are detailed in xref:client-configuration:registration-bootstrap.adoc[].
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:17]:
```suggestion
A bootstrap script can be created through the {webui} or from the command line. 
For more information about creating bootstrap script, see xref:client-configuration:registration-bootstrap.adoc[].
```

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 6]
```diff
@@ -3,87 +3,37 @@
...
+Remember to generate it with the activation key you've setup beforehand.
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:18]:
```suggestion
[NOTE]
====
Use  the activation key created before to generate the bootstrap script.
====
```

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 7]
```diff
@@ -3,87 +3,37 @@
...
-# NOTE: ACTIVATION_KEYS *must* be used to bootstrap a client machine.
-ACTIVATION_KEYS=1-migrate-salt
-ORG_GPG_KEY=
-----
-
-. When you have finished, save the file.
-
 [NOTE]
 ====
 By default, the bootstrap script will try to install [package]``venv-salt-minion`` for {salt} clients if it is available in the bootstrap repository, or [package]``salt-minion`` if there is no {salt} bundle in the bootstrap repository.
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:30]:
```suggestion
Bootstrap script will try to install [package]``venv-salt-minion`` for {salt} clients if it is available in the bootstrap repository, or [package]``salt-minion`` if there is no {salt} bundle in the bootstrap repository.
```

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 8]
```diff
@@ -3,87 +3,37 @@
...
-
-# can be edited, but probably correct (unless created during initial install):
-# NOTE: ACTIVATION_KEYS *must* be used to bootstrap a client machine.
-ACTIVATION_KEYS=1-migrate-salt
-ORG_GPG_KEY=
-----
-
-. When you have finished, save the file.
-
 [NOTE]
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:28]:
```suggestion
[IMPORTANT]
```

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 9]
```diff
@@ -3,87 +3,37 @@
...
 ====
 By default, the bootstrap script will try to install [package]``venv-salt-minion`` for {salt} clients if it is available in the bootstrap repository, or [package]``salt-minion`` if there is no {salt} bundle in the bootstrap repository.
-It is posible to avoid installing {salt} bundle and keep using [package]``salt-minion`` if you need it.
+It is possible to avoid installing {salt} bundle and keep using [package]``salt-minion`` if you need it.
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:32]:
```suggestion


```

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 10]
```diff
@@ -3,87 +3,37 @@
...
 
 For more information, see xref:client-configuration:contact-methods-saltbundle.adoc[Salt Bundle].
 ====
 
 
-
 == Run bootstrap script
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:37]:
```suggestion

== Run Bootstrap Script
```

**Review by 0rnela — APPROVED** [review]
Great work. Thanks a lot @vandabarata.

**Pull request merged**